.packit.yaml

```
@@ -36,3 +40,70 @@ jobs:
       # do not get the version from a tag (git describe) but from the spec file
       get-current-version:
         - grep -oP '^Version:\s+\K\S+' packaging/command-line-assistant.spec
+
+  # Test jobs
+  - &tests-e2e-vm
+    job: tests
+    identifier: vm-e2e-tests
```

```suggestion
  - &upstream-cla-tests-default
    job: tests
    identifier: upstream-cla-tests-defaults
```

It's hard to find a descriptive name, here's my attempt :smile:

---

I agree, that "e2e-vm" is not very descriptive, however "upstream-cla" feels unnecessary.
I mean, everyone understands, that this is an upstream part of the "cla" project.
Even the "tests" part might be a no-op.
If we take the whole task name `testing-farm:RHEL-9-Nightly-x86_64:container-sanity-tests`, then `testing-farm` is kind of implying that we're looking at a test task.
So I would stick with a descriptor of where (container/vm/cloud) it's running and what (suite) it's running.

---

I agree that having the "cla-upstream" there feels a bit useless (as of now). I changed the naming to have these two indetifiers:

- default-tests-virtual-machine
- sanity-tests-container
- sanity-tests-virtual-machine